src/webhook_payload_template.js

```diff
@@ -0,0 +1,164 @@
+const { JsonToken, jsonStringifyExtended } = require('./utilities.client');
```

Comment:
Wasn't this supposed to be called `JsonVariable` ?

Reply:
Don't know, that was @mtrunkat's PR. I'm just using it.

Reply:
I renamed that to `JsonVariable`